test/geotiff.spec.js

```diff
@@ -1,12 +1,25 @@
 import isNode from 'detect-node';
 import { expect } from 'chai';
+import http from 'http';
+import serveStatic from 'serve-static';
+import finalhandler from 'finalhandler';
 
-import { GeoTIFF, fromArrayBuffer, writeArrayBuffer, Pool } from '../src/geotiff';
+import { GeoTIFF, fromArrayBuffer, writeArrayBuffer, Pool, fromUrls } from '../src/geotiff';
 import { makeFetchSource, makeFileSource } from '../src/source';
 import { chunk, toArray, toArrayRecursively, range } from '../src/utils';
 import DataSlice from '../src/dataslice';
 import DataView64 from "../src/dataview64";
 
+// Set up a node server to make tiffs available at localhost:3000/test/data
+before(async (done) => {
+  const serve = serveStatic(__dirname);
+  const server = http.createServer(function onRequest (req, res) {
+    serve(req, res, finalhandler(req, res));
+  });
+  server.listen(3000);
+  done();
+});
```

Comment:
Hi @constantinius 
The new dependencies get used here to create a simple server for serving the tiffs in `test/data`.

Reply:
Ah, that makes sense.

Unfortunately, this seems to break the travis-ci build, as the static files server stalls the process, which seems to hang the tests.

Reply:
Grrr, I can't run the tests properly locally because of some weird things getting the full range of fixtures with the setup-data script. 
Strange that some of the tests run in Travis but yeah it then hangs....

Reply:
Turns out I needed to close the file server after the tests had completed.

Reply:
Awesome, thanks